src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -358,7 +350,6 @@ private string GetDisplayName(string httpMethod, string httpRoute = null)
             : $"{normalizedMethod} {httpRoute}";
     }
 
-#if !NETSTANDARD2_0
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     private void AddGrpcAttributes(Activity activity, string grpcMethod, HttpContext context)
```

Comment:
Could we mark this method static in this PR itself since it's related?

```suggestion
    private static void AddGrpcAttributes(Activity activity, string grpcMethod, HttpContext context)
```